File: AlgorithmsJS/InsertionSort.js
function insertionSort(list) {
  for (let index = 1; index < list.length; index++) {
    let position = index;
    let temp_value = list[position];

    while( position > 0 && list[position - 1] > temp_value ) {
      list[position] = list[position - 1];
      position--;
    }

    list[position] = temp_value;
  }

  return list;
}

module.exports = insertionSort;

File: AlgorithmsPY/insertion_sort.py
def insertion_sort(list):
  for index in range(1, len(list)):
    position = index
    temp_value = list[index]

    while position > 0 and list[position - 1] > temp_value:
      list[position] = list[position - 1]
      position = position - 1

    list[position] = temp_value

  return list
